static/openapi.yaml

```diff
@@ -0,0 +1,568 @@
+openapi: 3.0.2
```

Comment:
I noticed that this file was added in addition to the `openapi.yaml` that lives at the top level. I think its fine to delete the one at the top level if this is going to be the one served from now on. It'll only be confusing to keep both of them around.